.github/workflows/deployment.yml

```diff
@@ -23,10 +23,10 @@ jobs:
            displayName: macOS
            jpackageDownload: https://download.java.net/java/early_access/jdk14/27/GPL/openjdk-14-ea+27_osx-x64_bin.tar.gz
            jdk14Path: /jdk-14.jdk/Contents/Home
-           archivePortable: tar -czf build/distribution/JabRef-portable_macos.tar.gz -C build/distribution JabRef.app && rm -R build/distribution/JabRef.app
+           archivePortable: brew install pigz && tar -c -C build/distribution JabRef.app | pigz --rsyncable > build/distribution/JabRef-portable_macos.tar.gz && rm -R build/distribution/JabRef.app
 
     runs-on: ${{ matrix.os }}
-    name: Deploy on ${{ matrix.displayName }}
+    name: Build for ${{ matrix.displayName }}
```

Comment:
Is build the right terminology since we build the installer and not the application (compilation)?

Reply:
I would guess so as "build" (for me) is to create something. Be it the plain binary, the installer, ...

Reply:
I would guess so as "build" (for me) is to create something. Be it the plain binary, the installer, ...

Reply:
Create installer for matrix.os ...

Reply:
We also create the portable files. I will then use the longer name containing both.